gnome-base/gnome-control-center/gnome-control-center-43.0.ebuild

```diff
@@ -0,0 +1,195 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
+
+inherit gnome.org gnome2-utils meson python-any-r1 xdg
+
+DESCRIPTION="GNOME's main interface to configure various aspects of the desktop"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/gnome-control-center"
+SRC_URI+=" https://dev.gentoo.org/~mattst88/distfiles/${PN}-43.0-patchset.tar.xz"
+SRC_URI+=" https://www.gentoo.org/assets/img/logo/gentoo-horizontal.svg -> gentoo-gnome-control-center.svg"
```

Comment:
We probably shouldn't rely on this location never changing. I'll upload this to my distfiles and change SRC_URI.

Reply:
Yes, give me the URL and i will put it in.